### ISSUE TITLE
README possibly out of date

### ISSUE THREAD

**Issue by jajik** [modcluster/mod_cluster#787]
In the README, there are several comments such as

> Distribution archives are provided for each Tomcat version.

but I don't see any.

Does this still hold true? If yes, I would suggest clarifying (or even better adding links) , otherwise we should revise this.

**Comment by rhusar** [comment]
Yes, this is true - these are provided via maven, e.g. see https://repository.jboss.org/org/jboss/mod_cluster/mod_cluster-distribution/2.0.4.Final/

**Comment by rhusar** [comment]
Nevertheless - this should be better described better in the readme.

**Comment by rhusar** [comment]
Also opened https://github.com/modcluster/docs.modcluster.io/issues/190

**Comment by jajik** [comment]
Yes, that's what I meant. I haven't found any direct link. I'll try to propose something.
